packages/esm-billing-app/src/claims/claims-management/table/preauth-table.tmp.component.tsx

```diff
@@ -193,15 +206,32 @@ const PreauthTableTemporary: React.FC = () => {
 };
 
 export default PreauthTableTemporary;
-
-const ClaimStatus = ({ cell, row }: { cell: { value: string }; row: any }) => {
+const ClaimStatus = ({ row }: { row: any }) => {
```

Comment:
```suggestion
const ClaimStatus = ({ row }: { row: any }) => {
```
Please have the correct type here